src/index-fn.ts

```diff
@@ -1,3 +1,7 @@
+import { apply, isPolyfilled, isSupported } from './popover.js';
+
+export { apply, isSupported, isPolyfilled };
+
```

Comment:
```suggestion
```

Reply:
@jgerigmeyer I wrote that because I believed I needed to export those in order to use it in TS, is that not the case for this?

This is my first time implementing polyfills so pardon my ignorance on this. Im learning as I go.

Reply:
No apology necessary! Thanks for explaining. In this case the `export * from './popover.js';` at the bottom of the file exports _all_ functions from `popover.js`, so this isn't needed.

Reply:
@jgerigmeyer I see what you are saying. Ok ill commit your changes on my PR, would that be all?